spec/openapi.yaml

```diff
@@ -53,6 +53,34 @@ info:
       VAT is reverse charged, 0% tax is applied
 
     The tax strategy is derived based on order and store information and cannot be specified as input
+
+    ## Payment information
+    MyOnlineStore orders may be paid using a number of payment methods, through various gateways (providers).
+    A payment choice is uniquely specified by the combination of gateway (provider) and method (paytype).
+    MyOnlineStore also offers one internal gateway (`basic`) which may be used for manually verifiable payments
+    (for example: cash or card at the counter)
+
+    Orders may follow one of two payment flows:
+
+    1. The order has zero or more transactions. At this time the transaction amount will always equal the
+    full order amount (including tax), but partial payments may be added in the future. To be considered finished,
+    at least one transaction must have reached pending, authorized or captured status.
```

Comment:
Does this also apply to orders without a payment attached? According to this section, these orders can never be considered finished.

Reply:
No, orders without a payment attached are exempt from this whole payment section